### PR TITLE
{lib,tools,vis}[GCCcore/6.4.0] XZ v5.2.3, gettext v0.19.8.1, libxml2 v2.9.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2049.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a64f3745e07dd1f02df177803d9857d8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2443.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f610f01d5df739d4f6ff20a292868bde for a full test report.

File: easybuild/easyconfigs/g/gettext/gettext-0.19.8.1-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+homepage = 'http://www.gnu.org/software/gettext/'
+description = """GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may
+build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools
+and documentation"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/g/gettext/gettext-0.19.8.1-GCCcore-6.4.0.eb]: Missing checksum

File: easybuild/easyconfigs/l/libxml2/libxml2-2.9.4-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'http://xmlsoft.org/sources/',
+    'http://xmlsoft.org/sources/old/'
+]
+sources = [SOURCELOWER_TAR_GZ]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/l/libxml2/libxml2-2.9.4-GCCcore-6.4.0.eb]: Missing checksum

File: easybuild/easyconfigs/x/XZ/XZ-5.2.3-GCCcore-6.4.0.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+version = '5.2.3'
+
+homepage = 'http://tukaani.org/xz/'
+description = "xz: XZ utilities"
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['http://tukaani.org/xz/']
+
```
Comment (vanzod) [inline easybuild/easyconfigs/x/XZ/XZ-5.2.3-GCCcore-6.4.0.eb]: Missing checksum

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2049.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b8e7631c3571774bfd779f057c9a66fd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2443.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/03660eb4ec27c5fcb3fb66aaee0748d2 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**